docs/introduction/dev-environment-quick-start-guide.md

```diff
@@ -51,22 +61,24 @@ cd openfoodfacts-server/
-* `PRODUCT_OPENER_FLAVOR` can be modified to run different flavors of OpenFoodFacts, amongst `openfoodfacts` (default), `openbeautyfacts`, `openpetfoodfacts`, `openproductsfacts`.
-* `PRODUCT_OPENER_FLAVOR_SHORT` can be modified to run different flavors of OpenFoodFacts, amongst `off` (default), `obf`, `oppf`, `opf`.
-* `PRODUCERS_PLATFORM` can be set to `1` to build / run the **producer platform**.
-* `ROBOTOFF_URL` can be set to **connect with a Robotoff instance**.
-* `GOOGLE_CLOUD_VISION_API_KEY` can be set to **enable OCR using Google Cloud Vision**.
-* `CROWDIN_PROJECT_IDENTIFIER` and `CROWDIN_PROJECT_KEY` can be set to **run translations**.
-* `GEOLITE2_PATH`, `GEOLITE2_ACCOUNT_ID` and `GEOLITE2_LICENSE_KEY` can be set to **enable Geolite2**.
-* `TAG` is set to `latest` by default, but you can specify any Docker Hub tag for the `frontend` / `backend` images. Note that this is useful only if you use pre-built images from the Docker Hub (`docker/prod.yml` override); the default dev setup (`docker/dev.yml`) builds images locally.
+| Field | Description |
+| ----------------------------------------------------------------- | --- |
+| `PRODUCT_OPENER_DOMAIN`                                           | Can be set to different values based on which **OFF flavor** is run.|
+| `PRODUCT_OPENER_PORT`                                             | can be modified to run NGINX on a different port. Useful when running **multiple OFF flavors** on different ports on the same host. <br> Default port: `80`.|
+| `PRODUCT_OPENER_FLAVOR`                                           | Can be modified to run different flavors of OpenFoodFacts, amongst `openfoodfacts` (default), `openbeautyfacts`, `openpetfoodfacts`, `openproductsfacts`.|
+| `PRODUCT_OPENER_FLAVOR_SHORT`                                     | can be modified to run different flavors of OpenFoodFacts, amongst `off` (default), `obf`, `oppf`, `opf`.|
+| `PRODUCERS_PLATFORM`                                              | can be set to `1` to build / run the **producer platform**.|
+| `ROBOTOFF_URL`                                                    | can be set to **connect with a Robotoff instance**.|
+| `GOOGLE_CLOUD_VISION_API_KEY`                                     | can be set to **enable OCR using Google Cloud Vision**.|
+| `CROWDIN_PROJECT_IDENTIFIER` and `CROWDIN_PROJECT_KEY`            | can be set to **run translations**.|
+| `GEOLITE2_PATH`, `GEOLITE2_ACCOUNT_ID` and `GEOLITE2_LICENSE_KEY` | can be set to **enable Geolite2**.|
+| `TAG`                                                             | Is set to `latest` by default, but you can specify any Docker Hub tag for the `frontend` / `backend` images. <br> Note that this is useful only if you use pre-built images from the Docker Hub (`docker/prod.yml` override); the default dev setup (`docker/dev.yml`) builds images locally|
```

Comment:
I'm not really keen on table in markdown as it is difficult to maintain. Do you think it's really worth it ?

Reply:
It's far more legible IMO and usually docker images have them listed as a table: https://hub.docker.com/r/linuxserver/nextcloud#Parameters